action.yml

```diff
@@ -11,7 +11,7 @@ inputs:
     description: 'Path to a previously taken snapshot, if does not exists we will create it'
     default: '/tmp/default.snap'
   runtime:
-    description: 'Path to a try-runtime enabled runtime to test migrations for'
+    description: 'Path to a try-runtime enabled runtime to test migrations for, Set to SKIP to skip test and only download snapshot'
     required: true
```

Comment:
You can change this to `false` if you want to make this option optional

Reply:
There is a complication in how it is transferred to the Dockers' entrypoint with `$1 $2 $3 $4`